submissions/christopherUshie/html-form/contactForm.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<h1>The Contact form</h1>
+
+<form action="">
+    <fieldset>
+ 
+        <label for="fullname">Full name:</label>
+        <input type="text" id="fullname" name="fname"><br><br>
+        <label for="email">Email:</label>
+        <input type="email" id="email" name="email"><br><br>
+        <label for="subject">Subject:</label>
+        <input type="text" id="subject" name="subject"><br><br>
+        <label for="message">Message:</label>
+        <input type="text" id="message" name="message"><br><br>
```

Comment:
The input type should be a textarea tag and you should specify the rows and columns so as to accommodate the message.